packages/react-relay/modern/ReactRelayRefetchContainer.js

```diff
@@ -225,6 +234,10 @@ function createContainerWithFragments<TBase: ReactClass<*>>(
         }
         // TODO t15106389: add helper utility for fetching more data
         this._pendingRefetch = null;
+        this._relayContext = {
+          environment: this.context.relay.environment,
+          variables: fragmentVariables
```

Comment:
Just want to add that I was running this branch locally and it was the last [commit](https://github.com/facebook/relay/pull/1702/commits/d3c2acd4b589a76948c2657b0bec55175f017bdc) that assigned to `this._relayContext` that broke my application code. It caused my pagination component, that was wrapped by a refetch component, to receive no props after a successful GraphQL query on the initial render.